### PR TITLE
Changing isNaN to eqNaN

### LINKED ISSUE #977: isNaN('blue') is false
isNaN seems broken: http://bit.ly/1xirZyx
`R.isNaN('blue')` returns false

The fun thing is that if you "install" all functions in ramda to `window` to avoid having to type `R.` all the time, `R.isNaN` will replace window.isNaN and all libraries that uses isNaN might behave a bit irrationally :)

### PR DESCRIPTION
**Pull request by CrossEye**
Closes #977

### CODE REVIEW

File: dist/ramda.js [hunk 1]
```diff
@@ -932,9 +932,9 @@
      * @return {*} the first argument if falsy otherwise the second argument.
      * @example
      *
-     *      R.and(false, true); //=> true
-     *      R.and(0, []); //=> []
-     *      R.and(null, ''); => false
+     *      R.and(false, true); //=> false
+     *      R.and(0, []); //=> 0
+     *      R.and(null, ''); => null
```
Comment (davidchambers) [inline dist/ramda.js]: Don't commit changes to this file.

**Comment by davidchambers** [comment]
I like the current name. JavaScript has `isNaN` and `Number.isNaN`, and `R.isNaN` is clearly a member of the family. I don't find the argument for renaming the function compelling, but if we do decide to avoid naming collisions I suggest removing the function and suggesting people use `R.eq(NaN)` instead.

File: dist/ramda.js (hunk 1 above)
Comment (CrossEye) [inline dist/ramda.js]: How the hell did that get back in there?

**Comment by buzzdecafe** [comment]
removing and using `eq(NaN)` looks good to me

**Comment by CrossEye** [comment]
That would be fine with me too.  This function is a little specialized in any case.  Since `eq` is actually built to handle this, I think that may be best.

**Comment by CrossEye** [comment]
Will open another one soon to remove the function and suggest `eq(NaN)`.  I suppose with a deprecation first.

**Pull request closed without merging**